src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -380,9 +388,87 @@ You can also query for all existing streams. Here's an example:
           nrql
           status
           updatedAt
+          payloadCompression
         }
       }
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Export Compression [#compression]
+
+Optionally, we can compress payloads before they are exported, though this is disabled by default. This can help avoid hitting your ingested data limit and save money on egress COGS.
+
+You can enable compression using the `payloadCompression` field under `ruleParameters`. This field can be any of the following values:
+
+* `DISABLED`: Payloads will not be compressed before being exported. If unspecified, `payloadCompression` will default to this value.
+* `GZIP`: Compress payloads with the GZIP format before exporting
```

Comment:
```suggestion
Optionally, you can compress payloads before they are exported to save money on egress COGS. Compression is disabled by default. This can help avoid hitting your ingested data limit and save money on egress COGS.

You can enable compression using the `payloadCompression` field under `ruleParameters`. This field can be any of the following values:

* `DISABLED`: Payloads will not be compressed before being exported. If unspecified, `payloadCompression` will default to this value.
* `GZIP`: Compress payloads with the GZIP format before exporting
```

Reply:
If we make this change, we should move the `Compression is disabled by default.` sentence to the end of the paragraph for the sake of clarity. We might also want to remove the redundant `and save money on egress COGS` at the end.